jsk_fetch_robot/jsk_fetch.rosinstall

```diff
@@ -1,25 +1,67 @@
 # This file is for work space in fetch PC.
 
-# For proximity sensor on fetch hand.
+# To install app_manager.launch
+# remove after 1.1.0 (https://github.com/PR2/app_manager/commit/7fa0929b6608246073a0fb89eed013edd6b934ca) is released
 - git:
-    local-name: FA-I-sensor
-    uri: https://github.com/RoboticMaterials/FA-I-sensor.git
-    version: master
-# TODO(furushchev): describe why this is needed.
+    local-name: PR2/app_manager
+    uri: https://github.com/PR2/app_manager.git
+    version: 1.1.0
+# For fetch to use twitter
 - git:
-    local-name: ros-drivers/audio_common
-    uri: https://github.com/furushchev/audio_common.git
-    version: device
-# TODO(furushchev): describe why this is needed.
+    local-name: furushchev/image_pipeline
+    uri: https://github.com/furushchev/image_pipeline.git
+    version: develop
+# To send lifelog data to musca
+- git:
+    local-name: furushchev/mongodb_store
+    uri: https://github.com/furushchev/mongodb_store.git
+    version: develop
```

Comment:
`mongodb_store` is now released.
PR2 now uses `0.4.4` and works fine.

Reply:
Thank you for your information.

I pushed commit.